### ISSUE TITLE
Fix default database config not being created

### ISSUE THREAD

**Issue by NotJoeMartinez** [NotJoeMartinez/yt-fts#90]
on macos/linux default config path should be 
```python
 db_path = f"{os.path.join(os.getenv('HOME'), '.config', 'yt-fts')}/subtitles.db"
```
on windows 
```python
db_path = f"{os.path.join(os.getenv('APPDATA'), 'yt-fts')}/subtitles.db"
```
for some reason it's defaulting to the current directory

**Comment by NotJoeMartinez** [comment]
fixed in: "Fixed bootstrapping config dir and directory #91 "

**Comment by NotJoeMartinez** [comment]
fixed in: "Fixed bootstrapping config dir and directory https://github.com/NotJoeMartinez/yt-fts/pull/91 "
